tests/device_tests/test_msg_recoverydevice_shamir_dryrun.py

```diff
@@ -54,7 +54,7 @@ def input_flow():
             label="label",
             language="english",
             dry_run=True,
-            type=messages.ResetDeviceBackupType.Slip39_Single_Group,
+            type=messages.BackupType.Slip39_Basic,
```

Comment:
this is actually an inappropriate value, `type` is "matrix" or "scrambled words", only relevant on T1. so please remove this line